### PR TITLE
DOP-1732: add published branches file for commandline tools

### PR DESCRIPTION
**Pull request by schmalliso**

### CODE REVIEW

File: publishedbranches/docs-commandline-tools.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+prefix: '/'
```
Comment (madelinezec) [inline publishedbranches/docs-commandline-tools.yaml:1]: I believe this should be 'tools'

**Review by madelinezec — CHANGES_REQUESTED** [review]

File: publishedbranches/docs-commandline-tools.yaml (hunk 1 above)
Comment (madelinezec) [reply publishedbranches/docs-commandline-tools.yaml:1]: or 'database-tools' (depending on which site hosts the content of this repo) -- but certainly not '/' :)

Comment (schmalliso) [reply publishedbranches/docs-commandline-tools.yaml:1]: oh lord, of course. cloud-docs's prefix is `/` because it's at its own weird url, right?

**Review by madelinezec — APPROVED** [review]

File: publishedbranches/docs-commandline-tools.yaml (hunk 1 above)
Comment (madelinezec) [reply publishedbranches/docs-commandline-tools.yaml]: yes exactly!!

**Pull request merged**